frontend/src/lib/components/hedgehogs.tsx

```diff
@@ -0,0 +1,87 @@
+// Loads custom icons (some icons may come from a third-party library)
+import React, { ImgHTMLAttributes } from 'react'
+import surprisedHog from 'public/hedgehog/surprised-hog.png'
+import xRayHog from 'public/hedgehog/x-ray-hog.png'
+import hospitalHog from 'public/hedgehog/hospital-hog.png'
+import blushingHog from 'public/hedgehog/blushing-hog.png'
+import laptopHog1 from 'public/hedgehog/laptop-hog-01.png'
+import laptopHog2 from 'public/hedgehog/laptop-hog-02.png'
+import explorerHog from 'public/hedgehog/explorer-hog.png'
+import runningHog from 'public/hedgehog/running-hog.png'
+import spaceHog from 'public/hedgehog/space-hog.png'
+import tronHog from 'public/hedgehog/tron-hog.png'
+import heartHog from 'public/hedgehog/heart-hog.png'
+import starHog from 'public/hedgehog/star-hog.png'
+import policeHog from 'public/hedgehog/police-hog.png'
+import sleepingHog from 'public/hedgehog/sleeping-hog.png'
+import builderHog1 from 'public/hedgehog/builder-hog-01.png'
+import builderHog2 from 'public/hedgehog/builder-hog-02.png'
+import builderHog3 from 'public/hedgehog/builder-hog-03.png'
+import professorHog from 'public/hedgehog/professor-hog.png'
+import supportHeroHog from 'public/hedgehog/support-hero-hog.png'
+import xRayHog2 from 'public/hedgehog/x-ray-hogs-02.png'
+import laptopHog3 from 'public/hedgehog/laptop-hog-03.png'
+
+export const SurprisedHog = (props: ImgHTMLAttributes<HTMLImageElement>): JSX.Element => {
+    return <img src={surprisedHog} width={200} height={200} {...props} />
+}
+export const XRayHog = (props: ImgHTMLAttributes<HTMLImageElement>): JSX.Element => {
+    return <img src={xRayHog} width={200} height={200} {...props} />
```

Comment:
the idea `width={200} height={200}` is represented many times in the same file

I wonder if it's worth having an unexported `BaseHog` that sets the width and height to make it clear that individual hogs don't get to set width and height